### ISSUE TITLE
Validating a transcription causes error

### ISSUE THREAD

**Issue by cdausmus** [AtlasOfLivingAustralia/volunteer-portal#406]
A camera trap project (Possums in the rain!? (HJ_F03_Batch03)) has a number of transcriptions that cannot be validated due to an error occurring.

Error:
```
java.lang.NullPointerException: Cannot get property 'vernacularName' on null object
        at gsp_volunteer_portal_transcribe_wildlifeSpotterWidget_gsp.run(gsp_volunteer_portal_transcribe_wildlifeSpotterWidget_gsp.groovy:28)
```

**Comment by cdausmus** [comment]
@AdamWoodsAM 
Took all day but I found the culprit. One of the transcriptions includes a value no longer a valid option in the list of animals in the list. As a result, a (grails) template was being called with a null parameter.

The immediate fix would be to delete the transcription. I'll commit a code change that will skip over transcriptions that contain answers no longer possible (i.e. they were removed after activation).
